node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
> Actually, I might even consider it a bit strange to have is* for getting and * for setting.

I believe the use of functions as a nice convention for making an action in reference to the LED. Something like `isOn()` is saying, "Tell me if the LED is on or not."

However, if we were to use defined properties, like [value](https://github.com/tessel/t2-firmware/pull/110/files#diff-84087272b70af9498872cf6f791003abR885), instead of functions on the LED prototype, I would try to simplify the state property to only use `.on`:

```js
var led = tessel.led[0];

// returns true or false, would be false by default
console.log(led.on);

led.on = true; // Now the light is on

// returns true
console.log(led.on);
```

Having both `.on` and `.off` as defined properties can become confusing for how to control the LED. Using defined properties instead of functions can make managing the state of the LED throughout a program kind of confusing. For example:

```js
var led = tessel.led[0];

// isOn should probably have the value of false 
var isOn = led.on;

// Now I change the state of the LED
led.on = true;

// isOn will still return false but led.on returns true
console.log(isOn, led.on);
```

This might be a weird juggling act for some developers.